Vector.c

```diff
@@ -87,14 +88,22 @@ void Vector_prune(Vector* this) {
    this->items = 0;
 }
 
-static int comparisons = 0;
+//static int comparisons = 0;
+
+static void swap(Object** array, int indexA, int indexB) {
```

Comment:
Could as well be marked `inline` …

Reply:
Optimizers should do the *right* thing on static/file-local functions